### ISSUE TITLE
Shortcodes don’t parse variables

### ISSUE THREAD

**Issue by paulrobertlloyd** [11ty/eleventy#1089]
**Describe the bug**
Short codes only appear to accept strings for arguments. Passing a variable returns `undefined`.

**To Reproduce**

1. Register a new universal (or Liquid, same result) shortcode:

    ```js
    eleventyConfig.addShortcode('test', arg => {
      console.log('argument', arg);
    });
    ```

2. Use this shortcode with a string and a variable (Liquid template shown):

    ```
    {% test 'foobar' %}
    
    {% assign foobar = 'foobar' %}
    {% test foobar %}
    ```

3. Build your site. The following is output:

    ```bash
    argument foobar
    argument undefined
    ```

**Expected behaviour**
I expected to be able to pass variables to a shortcode, and for these to be parsed as strings.

**Environment:**

- OS and Version: macOS 10.15.4
- Eleventy Version: 0.11.0-beta.3

**Additional context**
Not sure if this is a regression in 0.11.beta-3, or if this has always been the case, as have only started experimenting with shortcodes.

**Comment by paulrobertlloyd** [comment]
@zachleat My apologies. I had a very insightful thought in the shower this morning; the bugs I’m finding in 11ty all relate to Liquid, and I’m using my own LiquidJS instance (v9.x). Sure enough, downgrading Liquid to use the built in version means the above issue no longer presents itself.

Linking this to PR #1058 as something to check when this upgrade is merged in.

**Comment by stefanfrede** [comment]
I ran into exactly the same problem today, I'm unable to pass down variables as arguments to my shortcodes. They all resolve to `undefined`.

I too use my own LiquidJs instance.

Is there something I can do to solve this?

**Comment by zachleat** [comment]
I tested this as fixed on both 1.0 and 2.0!

I’m guessing this was a duplicate of https://github.com/11ty/eleventy/pull/2367 in retrospect (shipped with 1.0.2 and 2.0.0-canary.9)

Thanks!
